### ISSUE TITLE
Inconsistency with keylines on 'Discussion' labels

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#6207]
**What**

Some of the ‘Discussion’ labels have a keyline some don’t

**Where**

https://wellcomecollection.org/event-series/WoK-NioAACkANeWK

Safari on both iOS and macOS but not Chrome nor Firefox.

Reported at: 
https://wellcome.slack.com/archives/C294K7D5M/p1615370133125800

https://wellcome.slack.com/archives/C294K7D5M/p1615221051082000

**Comment by pollecuttn** [comment]
![Screenshot 2021-03-08 at 16 28 20](https://user-images.githubusercontent.com/1446072/110923796-30bc4d00-8319-11eb-8cbf-ff8def4047d1.png)

**Comment by jennpb** [comment]
Safari / iOS 
![Library Insights  Wellcome Collection](https://user-images.githubusercontent.com/1446139/111194372-0ab1d980-85b3-11eb-95f9-e4537bbe29b8.png)

**Comment by jennpb** [comment]
Also reported by Ben on 23/4 https://wellcome.slack.com/archives/C294K7D5M/p1619182859015200

**Comment by alexwlchan** [comment]
I think this is history related.

In particular, after Ben's report, I noticed that he and I had keylines around different boxes within a serial. Why could that be?

I also noticed that I saw key lines appearing around the "Serial" links after I visited the individual stories – as I went down the page, the links gradually turned black. e.g. here I've opened parts 4 and 5, but not part 6:

<img width="673" alt="Screenshot 2021-04-23 at 14 14 30" src="https://user-images.githubusercontent.com/301220/115876468-422b6580-a43e-11eb-984b-d0068ece2c9f.png">

If I open that page in a private browsing window, I don't get the bug.

If I toggle the "Visited" style on the `<a>` tag in Safari, I can reliably trigger the bug. I've attached a screen capture.

https://user-images.githubusercontent.com/301220/115877073-f3320000-a43e-11eb-8afc-625478256494.mov
